mozilla-release/toolkit/mozapps/extensions/internal/XPIInstall.jsm

```diff
@@ -508,6 +508,20 @@ async function loadManifestFromWebManifest(aUri, aPackage) {
     addon.optionsBrowserStyle = manifest.options_ui.browser_style;
   }
 
+  // CLIQZ: Do not install addons which have omnibox or themes or is language pack
+  if (
```

Comment:
@tsl143 Could you please format this code to look like a FF style way?

`if ((manifest.omnibox && manifest.omnibox.keyword) ||`
     `(manifest.permissions && manifest.permissions.includes('theme') ||`
    `addon.type == 'webextension-langpack') {`
`...`
`}`